File: SA.Web/src/poco/GdprRecordDto.ts
import { GdprApplicationType } from '@/model';

export default interface GdprRecordDto {
    int: number;
    firstName: string;
    lastName: string;
    email: string;
    companyNumber: string;
    birtNumber: string;
    companyName: string;
    phoneNumber: string;
    street: string;
    city: string;
    postCode: string;
    countryId: number;
    description: string;
    type: GdprApplicationType;
    isProcessed: boolean;
}

File: SA.Application/Records/GdprRecordDto.cs
using SA.Core.Enums;

namespace SA.Application.Records
{
    public class GdprRecordDto
    {
        public virtual int Id { get; set; }
        public virtual string FirstName { get; set; }
        public virtual string LastName { get; set; }
        public virtual string Email { get; set; }        
        public virtual string CompanyNumber { get; set; }        
        public virtual string BirthNumber { get; set; }
        public virtual string CompanyName { get; set; }
        public virtual string PhoneNumber { get; set; }        
        public virtual string Street { get; set; }        
        public virtual string City { get; set; }        
        public virtual string PostCode { get; set; }        
        public virtual int CountryId { get; set; }        
        public virtual string Description { get; set; }
        public virtual GdprApplicationType Type { get; set; }
        public virtual bool IsProcessed { get; set; }
    }
}
